proposals/specification/interop.ttl

```diff
@@ -230,6 +226,7 @@ rdfs:label "Terms for application interoperability in Solid"@en .
   a rdf:Property ;
   rdfs:comment "Links to instances of the shape tree in a data registration"@en ;
   rdfs:isDefinedBy <> ;
+  rdfs:domain :DataRegistry ;
```

Comment:
We've not been using this in the DataRegistry of late (relying on ldp:contains instead). We do use this in Access Needs, Data Consents, and Data Grants though.